bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcTableModel.scala

```diff
@@ -23,15 +23,19 @@ object ItcTableModel {
   */
 sealed trait ItcTableModel extends AbstractTableModel {
 
-  /** Defines a set of header columns for all tables. */
-  val Headers = Seq(
-    Column("Data Labels",     (c, r) => (resultIcon(r).orNull, c.labels)),
-    Column("Images",          (c, r) => s"${c.count}",                      tooltip = "Number of exposures used in S/N calculation"),
-    Column("Exposure Time",   (c, r) => f"${c.singleExposureTime}%.1f",     tooltip = "Exposure time of each image [s]"),
-    Column("Total Exp. Time", (c, r) => f"${c.totalExposureTime}%.1f",      tooltip = "Total exposure time [s]"),
-    Column("Source Mag",      (c, r) => sourceMag(r),                       tooltip = "Source magnitude [mag]"),
-    Column("Source Fraction", (c, r) => sourceFraction(r),                  tooltip = "Fraction of images on source")
-  )
+  /// Define some generic columns. Values are rendered as strings in order to have them left aligned, similar to other sequence tables.
+  val LabelsColumn  = Column("Data Labels",     (c, r) => (resultIcon(r).orNull, c.labels))
+  val ImagesColumn  = Column("Images",          (c, r) => s"${c.count}",                      tooltip = "Number of exposures used in S/N calculation")
+  val CoaddsColumn  = Column("Coadds",          (c, r) => s"${c.coadds.getOrElse(1.0)}",      tooltip = "Number of coadds")
+  val ExpTimeColumn = Column("Exposure Time",   (c, r) => f"${c.singleExposureTime}%.1f",     tooltip = "Exposure time of each image [s]")
+  val TotTimeColumn = Column("Total Exp. Time", (c, r) => f"${c.totalExposureTime}%.1f",      tooltip = "Total exposure time [s]")
+  val SrcMagColumn  = Column("Source Mag",      (c, r) => sourceMag(r),                       tooltip = "Source magnitude [mag]")
+  val SrcFracColumn = Column("Source Fraction", (c, r) => sourceFraction(r),                  tooltip = "Fraction of images on source")
+
+
+  // Define different sets of columns as headers
+  val Headers           = Seq(LabelsColumn, ImagesColumn, ExpTimeColumn, TotTimeColumn, SrcMagColumn, SrcFracColumn)
+  val HeadersWithCoadds = Seq(LabelsColumn, ImagesColumn, CoaddsColumn, ExpTimeColumn, TotTimeColumn, SrcMagColumn, SrcFracColumn)
```

Comment:
It's best to use a concrete type like `Vector` or `List` rather than `Seq` because that type doesn't tell us very much (in particular a `Seq` can be mutable and/or infinite).